src/hotspot/cpu/x86/nativeInst_x86.hpp

```diff
@@ -428,6 +465,7 @@ class NativeLoadAddress: public NativeMovRegMem {
 
 // destination is rbx or rax
 // mov rbx, [rip + offset]
+//FIXME: Currently not being used in hotspot code base, extend it to support REX2 prefix.
 class NativeLoadGot: public NativeInstruction {
```

Comment:
Leftover from JAOTC implementation. There are few other methods we forgot to remove.
You don't need to fix it. Please, remove the comment.